<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CartesianGridDialog</class>
 <widget class="QDialog" name="CartesianGridDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>321</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="gridTypeGBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>2</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Grid Type</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="gridTypeVLay">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QRadioButton" name="gridTypeCartesianRBt">
       <property name="text">
        <string>Cartesian</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="gridTypeCylindRBt_2">
       <property name="text">
        <string>Cylindral</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="kdirGBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>K Direction</string>
   </property>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>52</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="kDirVLay">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QRadioButton" name="kUpDirRBt">
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="kDownDirRBt">
       <property name="text">
        <string>Down</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gridNumGBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Numbers of Grid Blocks</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>271</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="gridNumHLay">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="niVLay">
       <item>
        <widget class="QLabel" name="niLabel">
         <property name="text">
          <string>x direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="niLEdt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="njVLay">
       <item>
        <widget class="QLabel" name="njLabel">
         <property name="text">
          <string>y direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="njLEdt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="nkVLay">
       <item>
        <widget class="QLabel" name="nkLabel">
         <property name="text">
          <string>z direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nkLEdt"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gridOriginGBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Origin of Reservoir</string>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>271</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="gridOriginHLay">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="xOriginVLay">
       <item>
        <widget class="QLabel" name="xOriginLabel">
         <property name="text">
          <string>x direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="xOriginLEt">
         <property name="whatsThis">
          <string extracomment="where x starts with"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="yOriginVLay">
       <item>
        <widget class="QLabel" name="yOriginLabel">
         <property name="text">
          <string>y direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yOriginLEt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="zOriginVLay">
       <item>
        <widget class="QLabel" name="zOrignLabel">
         <property name="text">
          <string>z direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="zOriginLEt"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gridStepGBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Widths of Grid Blocks</string>
   </property>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>271</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="gridStepHLay">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="xStepVLay">
       <item>
        <widget class="QLabel" name="xStepLabel">
         <property name="text">
          <string>x direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="xStepLEt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="yStepVLay">
       <item>
        <widget class="QLabel" name="yStepLabel">
         <property name="text">
          <string>y direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yStepLEt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="zStepVLay">
       <item>
        <widget class="QLabel" name="zStepLabel">
         <property name="text">
          <string>z direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="zStepLEt"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>gridTypeCartesianRBt</tabstop>
  <tabstop>gridTypeCylindRBt_2</tabstop>
  <tabstop>kUpDirRBt</tabstop>
  <tabstop>kDownDirRBt</tabstop>
  <tabstop>niLEdt</tabstop>
  <tabstop>njLEdt</tabstop>
  <tabstop>nkLEdt</tabstop>
  <tabstop>xOriginLEt</tabstop>
  <tabstop>yOriginLEt</tabstop>
  <tabstop>zOriginLEt</tabstop>
  <tabstop>xStepLEt</tabstop>
  <tabstop>yStepLEt</tabstop>
  <tabstop>zStepLEt</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CartesianGridDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CartesianGridDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
